<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QMainWindow" name="Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Everpad</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="content"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="options">
      <item>
       <widget class="QComboBox" name="notebook">
        <property name="toolTip">
         <string>Select notebook</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tags">
        <property name="toolTip">
         <string>Note tags</string>
        </property>
        <property name="placeholderText">
         <string>Commas separated tags</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>351</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>Note</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionSave_and_close"/>
    <addaction name="actionDelete"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <addaction name="menuFIle"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_and_close">
   <property name="text">
    <string>Save and close</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
